Dockerfile

```diff
@@ -1,34 +1,19 @@
-FROM kbase/sdkbase2:python
-MAINTAINER KBase Developer
-# -----------------------------------------
-# In this section, you can install any system dependencies required
-# to run your App.  For instance, you could place an apt-get update or
-# install line here, a git checkout to download code, or run any other
-# installation scripts.
-
-# RUN apt-get update
-RUN pip install nbconvert==5.6.0 \
-    nbformat==4.4.0 \
-    traitlets==4.3.3 \
-    coverage==4.5.3 \
-    pytest-cov==2.8.1 \
-    pytest==4.0.2 \
-    flake8==3.5.0 \
-    coveralls==1.8.2 \
-    requests-mock==1.7.0 \
-    jsonrpcbase==0.2.0 \
-    python-dateutil==2.8.1
-
-# -----------------------------------------
-
-COPY ./ /kb/module
-RUN mkdir -p /kb/module/work
-RUN chmod -R a+rw /kb/module
+FROM python:3.10-alpine
+LABEL MAINTAINER KBase Developer
 
+COPY ./ /kb/module/
 WORKDIR /kb/module
+# install python modules one at a time so that all deps get resolved properly
+RUN pip install --upgrade pip && \
+    cat requirements.txt | sed -e '/^\s*#.*$/d' -e '/^\s*$/d' | xargs -n 1 pip install && \
+    cat requirements-test.txt | sed -e '/^\s*#.*$/d' -e '/^\s*$/d' | xargs -n 1 pip install && \
+    mkdir -p /kb/module/work && \
+    chmod -R a+rw /kb/module && \
+    cp compile_report.json work/
```

Comment:
copy the compile_report over to the work directory, which is where the Catalog expects it to be